website_sale_loyalty_suggestion_wizard/templates/promotion_templates.xml

```diff
@@ -0,0 +1,179 @@
+                                    id="promotion_config_label"
+                                    t-out="promotion_id.public_name"
+                                />
+                                <button
+                                    type="button"
+                                    class="btn-close"
+                                    data-bs-dismiss="modal"
+                                    aria-label="Close"
+                                />
+                            </div>
+                            <div class="modal-body" id="o_promo_config_body">
+                                <div
+                                    t-if="request.session.error_promo_code"
+                                    class="alert alert-danger"
+                                    role="alert"
+                                >
+                                    <span t-out="request.session.error_promo_code" />
+                                </div>
+                                <t
+                                    t-call="website_sale_loyalty_suggestion_wizard.configure"
+                                />
+                                <span
+                                    class="d-none js_promotion_change"
+                                    t-att-data-promotion-id="promotion_id and promotion_id.id or ''"
+                                />
+                                <span
+                                    class="d-none website_sale_order_id"
+                                    t-att-data-order-id="website_sale_order and website_sale_order.id or ''"
+                                />
+                            </div>
+                            <div class="modal-footer">
+                                <button
+                                    type="button"
+                                    class="btn btn-primary o_coupon_selection_wizard_apply"
+                                >Apply</button>
+                                <button
+                                    type="button"
+                                    class="btn btn-light"
+                                    data-bs-dismiss="modal"
+                                >Close</button>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </t>
+        </xpath>
+    </template>
+    <!-- List of suggested promotions in shopping cart -->
+    <template
+        id="website_sale_loyalty_suggestion"
+        inherit_id="website_sale.cart_lines"
+        active="True"
+        customize_show="True"
+        name="Promotions suggestions"
+    >
+        <xpath expr="//td[hasclass('td-img')]" position="before">
+            <t
+                t-set="promo"
+                t-value="line.suggested_promotion_ids.filtered(lambda x: x.is_published)[:1].id"
+            />
+            <td class="td-promo-suggestion" t-if="promo">
```

Comment:
Functionally tested and working, but this new conditional `<td>` breaks the UI. Please ensure a corresponding `<td>` is added in the header as well.

![image](https://github.com/user-attachments/assets/9bfb9e94-ae30-422c-a253-52fc268d3aeb)

Reply:
Done, thanks!

![image](https://github.com/user-attachments/assets/73bcb7cc-c4ff-4e8d-b6b6-3517ae4213fc)